packages/core/src/routes/log.ts

```diff
@@ -19,20 +20,33 @@ export default function logRoutes<T extends AuthedRouter>(
         userId: string().optional(),
         applicationId: string().optional(),
         logKey: string().optional(),
+        includeWebhookLogs: string().optional(),
       }),
       response: Logs.guard.omit({ tenantId: true }).array(),
       status: 200,
     }),
     async (ctx, next) => {
       const { limit, offset } = ctx.pagination;
       const {
-        query: { userId, applicationId, logKey },
+        query: { userId, applicationId, logKey, includeWebhookLogs: includeWebhookLogsString },
       } = ctx.guard;
+      const includeWebhookLogs = yes(includeWebhookLogsString);
 
       // TODO: @Gao refactor like user search
+      // Filter out webhook logs from the audit log list by default.
       const [{ count }, logs] = await Promise.all([
-        countLogs({ logKey, applicationId, userId }),
-        findLogs(limit, offset, { logKey, userId, applicationId }),
+        countLogs({
+          logKey,
+          applicationId,
+          userId,
+          includeWebhookLogs,
+        }),
+        findLogs(limit, offset, {
+          logKey,
+          userId,
+          applicationId,
+          includeWebhookLogs,
+        }),
```

Comment:
I think we can tell if the API is requesting webhook logs by just checking the logKey alone. Therefore it seems we don't necessarily have to add the new includeWebhookLogs param, no?

Reply:
`logKey` is optional and in most case we do not specify its value. It's decoupled with `includeWebhookLogs` IMO.